### DISCUSSION TITLE
text being sent from different account

### DISCUSSION THREAD

**Discussion by rcolepeterson** [speechly/speechly#317] [General]
Has anyone experienced "ghost text" appearing?
I have teammates complaining that speechly is sending text that when there is a silence.
My only thought is that text is coming in from a different account?

I told them that maybe a neighbor ... or something else is getting picked up, but i secretly think it is being sent by speechly from a diff account.

Thanks
Cole

**Comment by teelisyys** [comment]
Hi @rcolepeterson ! Thank you for using Speechly!

Due to architecture and implementation of our Streaming ASR, it is not possible for the results of different audio streams to mix.

We are aware that, especially when there is silence at the start of the audio stream, our ASR can pick up and recognise faint and even inaudible background speech from the audio stream.

Note that while some ASR system exhibit what is call "hallucination", i.e. producing seemingly coherent transcript out of nothing, this does not happen with our ASR system.

If your use case has an environment where faint background speech is expected, you might want to try our [Voice Activity Detection (VAD)](https://docs.speechly.com/basics/glossary#vad-voice-activity-detection) feature, which gates the audio at client side based on the energy level of the audio signal. It is available for most of our SDKs, including [browser-client](https://github.com/speechly/speechly/blob/main/libraries/browser-client/docs/interfaces/client.VadOptions.md) and [react-client](https://github.com/speechly/speechly/blob/main/libraries/react-client/docs/interfaces/context.SpeechProviderProps.md#vad).

**Reply by rcolepeterson** [reply]
Thank you! Makes sense. Love your code!
